src/main/java/net/sf/jabref/model/entry/KeywordList.java

```diff
@@ -0,0 +1,88 @@
+package net.sf.jabref.model.entry;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.StringTokenizer;
+import java.util.stream.Collectors;
+
+import net.sf.jabref.model.strings.StringUtil;
+
+/**
+ * Represents a list of keyword chains.
+ * For example, "Type > A, Type > B, Something else".
+ */
+public class KeywordList extends ArrayList<Keyword> {
```

Comment:
Composition over inheritance, noted!

I wasn't sure how to handle duplicates. There was some code which tried to replace a keyword exactly at the same position: `a, b, c` -> `a, replace, c`. This was not easily possible with Set, but just one line with List. Not sure that this was the right decision through. What is your opinion?